xl2times/transforms.py

```diff
@@ -2380,15 +2423,42 @@ def eval_and_update(
         table = model.commodity_groups
         updates = tables[datatypes.Tag.tfm_comgrp].filter(table.columns, axis=1)
 
-        commodity_groups = pd.concat(
-            [table, updates], ignore_index=True
-        ).drop_duplicates()
+        commodity_groups = pd.concat([table, updates], ignore_index=True)
+        commodity_groups["commodity"] = commodity_groups["commodity"].explode(
+            ignore_index=True
+        )
```

Comment:
```suggestion
        commodity_groups = commodity_groups.explode("commodity", ignore_index=True)
```

Reply:
This should address the regression on `COM_GMAP`

Reply:
/headsmack
Good spotting, thanks :)  All fixed!